arlas-core/src/main/java/io/arlas/server/utils/ParamsParser.java

```diff
@@ -66,27 +67,29 @@
     private static final List<OperatorEnum> GEO_OP_WITHIN = Arrays.asList(OperatorEnum.within, OperatorEnum.notwithin);
     private static final  GeometryFactory GEOMETRY_FACTORY = new GeometryFactory(new PrecisionModel(), 4326);
     private static final List<OperatorEnum> GEO_OP_INTERSECTS = Arrays.asList(OperatorEnum.intersects, OperatorEnum.notintersects);
```

Comment:
BTW, this constant does not appear to be used anywere

Reply:
removed